pylace/lace/utils.py

```diff
@@ -117,3 +122,46 @@ def infer_column_metadata(
         ).rename(column)
         mds.append(md)
     return mds
+
+
+_COMMON_TRANSITIONS = {
+    "sams": [
+        StateTransition.view_alphas(),
+        StateTransition.row_assignment(RowKernel.sams()),
+        StateTransition.view_alphas(),
+        StateTransition.row_assignment(RowKernel.sams()),
+        StateTransition.view_alphas(),
+        StateTransition.component_parameters(),
+        StateTransition.row_assignment(RowKernel.slice()),
+        StateTransition.component_parameters(),
+        StateTransition.column_assignment(ColumnKernel.gibbs()),
+        StateTransition.column_assignment(ColumnKernel.slice()),
```

Comment:
Gibbs can make small moves and also larger moves than slice, so we don't also need slice.

Reply:
Removed.